src/components/data/config.json

```diff
@@ -257,7 +264,7 @@
     "yaml":
     {
         "prolog":"# Wikibase with Query Service\n#\n# This docker-compose example can be used to build and run the images.\n#\n# Access Wikibase via 'http://localhost:8181'\n#   (or 'http://$(docker-machine ip):8181' if using docker-machine)\n#\n#   (or 'http://$(docker-machine ip):8282' if using docker-machine)\nversion: '3'\n\nservices:",
-        "wikibase": "    wikibase:\n        image: wikibase/wikibase:1.30-bundle\n        links:\n            - mysql\n        ports:\n            - \"{wikibasePort}:80\"\n        volumes:\n            - mediawiki-images-data:/var/www/html/images\n        depends_on:\n            - mysql{dependencyElasticsearch}\n        networks:\n            default:\n                aliases:\n                    - wikibase.svc\n        environment:\n            - DB_SERVER=mysql.svc:3306\n            - DB_USER={databaseUser}\n            - DB_PASS={databaseUserPassword}\n            - DB_NAME={databaseName}\n            - MW_ADMIN_NAME={mediawikiAdminName}\n            - MW_ADMIN_PASS={mediawikiAdminPassword}\n            - MW_WG_SECRET_KEY={secretkey}",
```

Comment:
Is there any way to make these strings more readable? it sucks that they have to be all on one line...

Reply:
perhaps we could split it in a subarray? So we build a kind of graph.